aspnetcore/web-api/http-repl.md

```diff
@@ -804,6 +804,88 @@ To set an HTTP request header, use one of the following approaches:
   https://localhost:5001/people~ set header Content-Type
   ```
 
+## Test secured endpoints
+
+The HTTP REPL supports the testing of secured endpoints through the use of HTTP request headers. Examples of supported authentication and authorization schemes include basic authentication, JWT bearer tokens, and digest authentication. For example, send a bearer token to an endpoint with the following command:
```

Comment:
> For example, send a bearer token to an endpoint with the following command:

This reads a little awkwardly - should it be either "... to send a bearer token ... use the following command:" or "For example, you can send a bearer token ..." or something similar?